ArcGISRuntimeSDKQt_QMLSamples/EditData/EditWithBranchVersioning/deployment.pri

```diff
@@ -0,0 +1,43 @@
+#-------------------------------------------------
+# Copyright 2020 Esri.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at:
+# http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#-------------------------------------------------
+
+
+android-no-sdk {
+    target.path = /data/user/qt
+    export(target.path)
+    INSTALLS += target
+} else:android {
+    x86 {
+        target.path = /libs/x86
+    } else: armeabi-v7a {
+        target.path = /libs/armeabi-v7a
+    } else {
+        target.path = /libs/armeabi
+    }
```

Comment:
Is this still needed with Qt 5.15? This is old logic.

Reply:
@JamesMBallard It definitely is not. I forgot to create an issue about this last time before I went on leave. I'll get a PR open to fix this.

Reply:
For the sake of keeping this sample moving forward, I just removed the unnecessary code in this case.

In regards to the broader issue of if it's necessary anymore, once I'm done looking into the logistics of that I will spin up another samples issue to have that fixed in previous samples as well since I would expect the vast majority of those to still have it since this template has been used for a while.